draft-ietf-quic-http.md

```diff
@@ -569,8 +578,9 @@ that is being cancelled (see {{frame-push-promise}}).
 If the client receives a CANCEL_PUSH frame, that frame might identify a Push ID
```

Comment:
There's a reference to a 32 bit Push ID three lines above this one.  I think that needs to be updated as well?

Reply:
Actually, there were two, thanks.

Reply:
Or three; I missed your example, but fixed two others...